go.mod

```diff
@@ -112,6 +112,8 @@ require (
 	cloud.google.com/go/compute v1.23.0 // indirect
 	cloud.google.com/go/compute/metadata v0.2.3 // indirect
 	cloud.google.com/go/iam v1.1.2 // indirect
+	github.com/DataDog/datadog-agent/pkg/util/log v0.49.0 // indirect
+	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.49.0 // indirect
```

Comment:
Scrubber comes because `log` uses it.

And `appsec-internal-go` uses `log` to report a bunch of debug entries about configuration loading, and reporting errors about invalid configuration found in environment variables...

Reply:
As discussed out of GH, I think this is an important issue to fix in appsec-internal-go, as we can't mix in dd-trace-go the agent logger [by mistake actually].

Reply:
Is this something that can be fixed before this code is merged?